appveyor.yml

```diff
@@ -81,7 +81,8 @@ install:
 
 - set PATH=C:\Python%PYTHON_VERSION%;C:\Python%PYTHON_VERSION%\Scripts;%PATH%
 - pip install nose pytest
-- pip install --install-option="--no-cython-compile" cython
+- if [%COMPILER%]==[MinGW-w64] pip install --install-option="--no-cython-compile" cython==0.26
```

Comment:
@certik, you were right. Downgrading cython fixed the error

Reply:
Yeah. It's better to pick a particular version of Cython, since they seem to break things for us from time to time.